governing-documents/security.md

```diff
@@ -0,0 +1,266 @@
+knowledge (e.g. in a particular area of cryptography) who is capable of
+fixing a problem in a short period of time.
+1. Each maintainer will be a member of the Hyperledger security email list
+(security@lists.hyperledger.org), and, in general, any Hyperledger-wide
+security infrastructure.
+2. If a project has specialized code related to certain aspects of security
+or cryptography (e.g. consensus algorithms or cryptographic algorithms), then
+a corresponding specialist should be on the response team (e.g. someone
+knowledgeable in consensus or cryptography, respectively).  If a specialist
+is not on the team, then the individual who is responsible for contacting or
+engaging the specialists should be designated in their stead.  We emphasize
+that projects should have access to specialists in an area for which they
+maintain code while recognizing that it may not be practical for these experts
+to be on the response team.
+
+The current response team maintainers are as follows:
+
+| Name | Email ID | Chat ID | Area/Specialty |
+|--- | --- | --- | ---|
+| Satoshi Nakamoto | satoshi@nsa.gov | theREALsatoshi | Everything |
+| Hart Montgomery | hart@donotmail.com | uselessIDIOT | Nothing |
+| <> | <> | <> | (if applicable) |
+
+**Discussion forum:**
+
+1. Projects may set up private discussion forums for approved maintainers and
+security team participants to discuss vulnerabilities.
+    1. The project uses the following private discussion forums for approved
+    maintainers and security team participants, which may include discord or
+    email lists:
+        1. <>
+        2. <>
+
+Forums may be private but the existence of a forum must be public.
+
+## Report Intakes
+While all Hyperledger Foundation projects are required to respond to security
+vulnerability disclosures sent to the Hyperledger security list
+(security@lists.hyperledger.org), many projects have other authorized ways
+that reports are taken in as well.
+
+This project has the following recommended ways to submit security
+vulnerabilities.  While the security team members will do their best to
+respond to bugs disclosed in all possible ways, it is encouraged for bug
+finders to report through the following approved channels:
+The Hyperledger Foundation security list:  security@lists.hyperledger.org.
+To report a security issue, please send an email with the name of the project,
+a description of the issue, the steps you took to create the issue, affected
+versions, and if known, mitigations for the issue.  Note that this is a
+mandatory response channel.  We encourage reporters to default to this channel
+for reporting vulnerabilities if there is no compelling reason to use other
+channels. Github security vulnerability reporting (see below for more github
+information):  https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability.
```

Comment:
Should the `see below for more github information` be a link to another section in this document? Or is it referring to this link given here?

Reply:
This part should have been pointers suggesting the possible options to report a security issue. The first one is the security mailing list, the other option is to use github's security reporting framework (the link following the text has more information on that). This section is acting as a template for project maintainers to define what they follow in their case.